artifacts/scripts/construct.sh

```diff
@@ -96,6 +96,32 @@ git config user.name "$GIT_COMMITTER_NAME"
 echo "Running garbage collection."
 git config gc.pruneExpire 3.days.ago
 git gc --auto
+
+# Remove corrupted lines in .git/packed-refs
+# Lines not starting with '^' caret are printed as-is. Lines
+# that do not start with a caret are printed ONLY if the previous
+# line did not start with a caret
+echo "Cleaning .git/packed-refs"
+cp packed-refs packed-refs.bak
+awk '
+NR == 1 {
+    prev_line_start_with_caret = 0
+    print
```

Comment:
This causes the first line (the comment `# pack-refs with: peeled fully-peeled sorted `  to be printed twice in the resulting file.

Reply:
fixed